venom/rpc/reflect/reflect.py

```diff
@@ -1,17 +1,44 @@
 from typing import Type, Set
 
 from venom import Message
-from venom.rpc import Method
+from venom.message import fields
+from venom.rpc.method import Method
+from venom.fields import Field, RepeatField, MapField
 from venom.rpc import Service
 
 
 class Reflect(object):
-    services: Set[Service]
-    methods = Set[Method]
-    messages: Set[Type[Message]]
+    services : Set[Service]
+    methods : Set[Type[Method]]
+    messages : Set[Type[Message]]
 
     def __init__(self):
         self.services = set()
+        self.methods = set()
+        self.messages = set()
 
-    def add(self, service: Type[Service]):
+    def _add_field(self, field: Field):
+        if type(field) == Field:
```

Comment:
Don't use `type(field) == Field`. Use `isinstance()`. Right now your code would fail with e.g. `ConverterField`. Your order and use of if statements may need to change.